### PR TITLE
KOGITO-5335 Remove the old TestUserSystem infrastructure from new web…

### PR DESCRIPTION
**Pull request by bxf12315**
resolves the bug https://issues.redhat.com/browse/KOGITO-5335

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

**Comment by pefernan** [comment]
jenkins retest this

File: ui-packages/packages/consoles-common/src/utils/KeycloakClient.ts [hunk 1]
```diff
@@ -19,10 +19,9 @@ import {
   ANONYMOUS_USER,
   User,
   UserContext,
-  TestUserContextImpl,
   KeycloakUserContext
 } from '../environment/auth';
-import { isTestUserSystemEnabled } from './Utils';
+// import { isTestUserSystemEnabled } from './Utils';
```
Comment (pefernan) [inline ui-packages/packages/consoles-common/src/utils/KeycloakClient.ts]: remove comment. Could you also remove Utils.ts file? It contains the `isTestUserSystemEnabled` and it's no longer needed.

File: ui-packages/packages/task-console-webapp/src/components/console/TaskConsole/tests/TaskConsole.test.tsx [hunk 2]
```diff
@@ -42,9 +42,11 @@ describe('TaskConsole tests', () => {
   it('Snapshot', () => {
     // @ts-ignore
     const client = new ApolloClientMock();
+    // @ts-ignore
+    const testUser = new TestUserContext();
```
Comment (pefernan) [inline ui-packages/packages/task-console-webapp/src/components/console/TaskConsole/tests/TaskConsole.test.tsx]: I know it's a test, but I think the variable could have a better name (`testContext` for example :))

File: ui-packages/packages/consoles-common/src/utils/tests/KeycloakClient.test.ts [hunk 3]
```diff
@@ -15,21 +15,12 @@
  */
 
 import * as KeycloakClient from '../KeycloakClient';
-import * as Utils from '../Utils';
+// import * as Utils from '../Utils';
```
Comment (pefernan) [inline ui-packages/packages/consoles-common/src/utils/tests/KeycloakClient.test.ts]: same here

**Review by pefernan — CHANGES_REQUESTED** [review]
@bxf12315 sorry for the late review... it's almost there... would you mind remove the unused imports and that would be all!

File: ui-packages/packages/task-console-webapp/src/components/console/TaskConsole/tests/TaskConsole.test.tsx (hunk 2 above)
Comment (bxf12315) [reply ui-packages/packages/task-console-webapp/src/components/console/TaskConsole/tests/TaskConsole.test.tsx]: done

File: ui-packages/packages/consoles-common/src/utils/KeycloakClient.ts (hunk 1 above)
Comment (bxf12315) [reply ui-packages/packages/consoles-common/src/utils/KeycloakClient.ts]: done

File: ui-packages/packages/consoles-common/src/utils/tests/KeycloakClient.test.ts (hunk 3 above)
Comment (bxf12315) [reply ui-packages/packages/consoles-common/src/utils/tests/KeycloakClient.test.ts]: done

**Review by pefernan — APPROVED** [review]
+1

**Review by Sara4994 — APPROVED** [review]

**Review by AjayJagan — APPROVED** [review]
LGTM :)

**Pull request merged**